app/controllers/api/v1/properties_controller.rb

```diff
@@ -20,7 +21,7 @@ def index
       end
 
       def filters
-        @areas = UnitRepository.get_filters
+        @areas = UnitRepository.get_filters.filter {|l| l != 'section #7 lot#106  las conchas' && l != 'los langostino, playa encanto'}
```

Comment:
```suggestion
        @areas = UnitRepository.get_filters.filter { |l| !l.in? ['section #7 lot#106  las conchas', 'los langostino, playa encanto'] }
```
Also, do these texts need to be hardcoded? (Just asking, because if they could change you could use an env var to avoid deploying text fixes)